app/workers/stale_issue_marker.rb

```diff
@@ -35,32 +35,18 @@ def pinned_labels
   #
   def process_stale_issues
     handle_newly_stale_issues
-    handle_stale_and_unmergable_prs
   end
 
   def handle_newly_stale_issues
-    query  = "is:open archived:false updated:<#{stale_date.strftime('%Y-%m-%d')}"
+    query  = "is:open archived:false -label:stale updated:<#{stale_date.strftime('%Y-%m-%d')}"
```

Comment:
I think if we remove the `-label:stale", then this will just comment every 3 months, which is probably what we want.